packages/petr4/petr4.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+synopsis: "Petr4: Formal Semantics for the P4 Programming Language"
+maintainer: "jnfoster@cs.cornell.edu"
+authors: ["Nate Foster <jnfoster@cs.cornell.edu>"]
+homepage: "https://github.com/cornell-netlab/petr4"
+dev-repo: "git+https://github.com/cornell-netlab/petr4/"
+bug-reports: "https://github.com/cornell-netlab/petr4/issues"
+license: "Apache2"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune"
```

Comment:
```suggestion
  "dune" {build}
```
This dependency is only needed at build time

Reply:
Thanks. Done.